crates/build/re_types_builder/src/codegen/rust/api.rs

```diff
@@ -1097,7 +1097,7 @@ fn quote_trait_impls_for_archetype(obj: &Object) -> TokenStream {
         #recommended_descriptors
         ComponentDescriptor {
             archetype_name: Some(#archetype_name.into()),
-            component_name: #indicator_name.into(),
+            component_name: #quoted_indicator_name::DEFAULT.descriptor().component_name,
```

Comment:
I would just go for this imo:
```suggestion
let indicator_fqname = format!("rerun.components.{}Indicator", obj.name);

Reply:
(that comment isn't in the right place, but you get the idea)

Reply:
that's unfortunately incorrect because the namespace may have blueprint in it and I couldn't be bothered putting that logic in here

Reply:
eh okay I do it anyways